homeassistant/util/color.py

```diff
@@ -186,7 +196,7 @@ class GamutType:
     blue: XYPoint = attr.ib()
 
 
-def color_name_to_rgb(color_name: str) -> tuple[int, int, int]:
+def color_name_to_rgb(color_name: str) -> RGBColor:
```

Comment:
This breaks the api compatibility, did you check that it's not a problem? From the looks of it, this util is only used by `light/__init__.py` and `light/intent.py`.

Reply:
I don't think it breaks anything as a `NameTuple` is essentially just a fixed length tuple itself.

Did a small test to confirm it. No errors when run with `mypy`:
```py

from typing import NamedTuple

class Color(NamedTuple):
    r: int
    g: int
    b: int

red = Color(255, 0, 0)

def func(hex_color: tuple[int, int, int]) -> None:
    pass

func(red)
```

--
It would be different if we start requiring `RGBColor` as argument somewhere, but that isn't the case currently.
```py
def func2(hex_color: Color) -> None:
    pass

func2((255, 0, 0))  # Argument 1 to "func2" has incompatible type
                    # "Tuple[int, int, int]"; expected "Color"
```

Reply:
Okay, makes sense, I learned something new today! Sorry for the noise and thanks for pointing this out! :-)